docs/sources/setup/install/logging-guide.md

```diff
@@ -0,0 +1,344 @@
+---
+title: Grafana Log Dashboard Guide
+menuTitle: Grafana Log Dashboard Guide
+description: Step-by-step guide for setting up Grafana, Loki, and Grafana Alloy for log aggregation and visualization.
+weight: 310
+---
+
+This guide provides an official, end-to-end reference for deploying a unified log monitoring system using Grafana, Loki, and Grafana Alloy. Centralized log collection is essential for rapid troubleshooting, security auditing, and system performance analysis. By automating both log transfer and ingestion, this setup minimizes operational overhead and ensures that logs from multiple sources arrive in Grafana in near real time.
+
+---
+
+## Step 1: Install Grafana via CLI
+
+Grafana is the visualization layer for metrics and logs. Installing via CLI guarantees reproducible deployments and supports automated provisioning in production environments.  
+* For complete installation instructions on all supported platforms, see the official Grafana guide: [Grafana Installation](https://grafana.com/docs/grafana/latest/setup-grafana/installation/)  
+* This example uses Ubuntu/Debian; if you’re on another OS, adapt the commands below as needed.
+
+```bash
+sudo apt-get update
+sudo apt-get install -y gnupg2 curl
+curl https://packages.grafana.com/gpg.key | sudo apt-key add -
+sudo add-apt-repository "deb https://packages.grafana.com/oss/deb stable main"
+sudo apt-get update
+sudo apt-get install -y grafana
+sudo systemctl start grafana-server
```

Comment:
Looks like you might be missing a step here?
https://grafana.com/docs/grafana/latest/setup-grafana/start-restart-grafana/

I'm a bit concerned that you've condensed what seems to be pages and pages in the Grafana installation documentation into a single copy/paste block.